src/common/components/share/twitter.js

```diff
@@ -0,0 +1,23 @@
+import React, { Component, PropTypes } from 'react';
+
+import styles from './styles.css';
+
+export class Tweet extends Component {
+  render() {
+    const { name } = this.props;
+    const text = encodeURIComponent(
+      `Install ${name} in seconds on Linux OSes:\nsudo snap install ${name}\n\n(Don’t have snapd? https://snapcraft.io/docs/core/install)`
```

Comment:
I know that right now we only use this to share install instructions, but if 'CopyToClipboard' gets text as param, maybe it would be good to make `Tweet` work the same way as well?

Also... are we sure that's the message we want to tweet? With the only link being to https://snapcraft.io/docs/core/install rather than any snap/build related page?